CONTRIBUTORS

```diff
@@ -31,6 +31,7 @@ Rohr, David
 Shahoyan, Ruben
 Stockmanns, Tobias
 Ustyuzhanin, Andrey
+Vilela, Cristovao
 von Haller, Barthélémy
 Wang, Yanzhao [https://orcid.org/0000-0002-7006-7986]
```

Comment:
BTW: If you have an [ORCID](https://orcid.org/), you're allowed to add it here like in the example two lines below.

Reply:
Oh, and can you please update the generated metadata from this? <https://github.com/FairRootGroup/FairRoot/actions/runs/10112922390/job/27969344166?pr=1576>

Reply:
@ChristianTackeGSI, do you want me to edit `codemeta.json` by hand?

Reply:
> do you want me to edit codemeta.json by hand?

If you want to? Otherwise we'll do it during the release management, which is fine as well.

If you do, please squash it into the CONTRIBUTORS commit and also update zenodo.json: 

https://github.com/FairRootGroup/FairRoot/actions/runs/10113766901/job/27975442182?pr=1576